symplyphysics/definitions/wave_equation_in_one_dimension.py

```diff
@@ -0,0 +1,38 @@
+from sympy import (
+    Derivative,
+    Eq,
+    symbols,
+    Function as SymFunction,
+)
+from symplyphysics import (
+    units,
+    Symbol,
+    print_expression,
+)
+
+# Description
+## The wave equation is a second-order linear partial differential equation used to
+## describe the description of waves, including standing wave fields such as mechanical
+## or electromagnetic waves.
+
+# Law: d**2(u(x, t))/dx**2 = (1/v**2) * d**2(u(x, t))/dt**2
+## u(x, t) - factor representing a displacement from rest situation
+## x - position
+## t - time
+## d**2/dx**2 - partial derivative w.r.t. position x
+## d**2/dt**2 - partial derivative w.r.t. time t
+## v - wave speed (a fixed non-negative real value)
```

Comment:
Is it the same thing, as in https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/waves/phase_velocity_from_angular_frequency_and_wavenumber.py ? Maybe we could share the same name?

Reply:
They are the same